docs/src/gettingstarted.md

```diff
@@ -0,0 +1,222 @@
+you would run:
+
+```@repl conda
+ENV["PATH"] = ENV["PATH"] * ":" * "/home/kevin/.local/bin"
+```
+
+If you don't know where your installation is located,
+from the terminal, you can use the `which` command:
+
+```sh
+$ which humann
+/home/kevin/.local/bin/humann
+```
+
+### Using Conda.jl
+
+If you don't have a previous installation, you can use `Conda.jl` to install the necessary tools.
+To do this, first install `Conda.jl` in your environment using the Pkg REPL
+(accessible by typing `]` in the julia REPL - press `<backspace>` to get back to the regular REPL).
+
+```plaintext
+$ cd my_project/
+
+$ julia --project
+               _
+   _       _ _(_)_     |  Documentation: https://docs.julialang.org
+  (_)     | (_) (_)    |
+   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
+  | | | | | | |/ _` |  |
+  | | |_| | | | (_| |  |  Version 1.6.1 (2021-04-23)
+ _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
+|__/                   |
+
+julia> # press ']'
+
+(my_project) pkg> add Conda
+    Updating registry at `~/.julia/registries/General`
+   Resolving package versions...
+    Updating `~/.julia/dev/BiobakeryUtils/my_project/Project.toml`
+  [8f4d0f93] + Conda v1.5.2
+    Updating `~/.julia/dev/BiobakeryUtils/my_project/Manifest.toml`
+  [8f4d0f93] + Conda v1.5.2
+  [682c06a0] + JSON v0.21.2
+  [69de0a69] + Parsers v2.0.4
+  [81def892] + VersionParsing v1.2.0
+  [ade2ca70] + Dates
+  [a63ad114] + Mmap
+  [de0858da] + Printf
+  [4ec0a83e] + Unicode
+
+julia> using Conda
+```
+
+First, you'll need to add "channels" to a new Conda environment.
+The order here is important.
+Assuming you want your environment to be called `biobakery`:
+
+```julia-repl
+julia> Conda.add_channel("bioconda", :biobakery)
+┌ Info: Running conda config --add channels bioconda --file /home/kevin/.julia/conda/3/envs/biobakery/condarc-julia.yml --force
+└ in biobakery environment
+# ...
+
+julia> Conda.add_channel("conda-forge", :biobakery)
+┌ Info: Running conda config --add channels conda-forge --file /home/kevin/.julia/conda/3/envs/biobakery/condarc-julia.yml
+└ --force in biobakery environment
+# ...
+
+julia> Conda.add("humann", :biobakery)
+[ Info: Running conda install -y -c bioconda humann in biobakery environment
+Collecting package metadata (current_repodata.json): done
+Solving environment: done
+# ...
+
+julia> Conda.add("metaphlan", :biobakery; channel="bioconda")
+[ Info: Running conda install -y -c bioconda metaphlan in biobakery environment
+Collecting package metadata (current_repodata.json): done
+Solving environment: done
+# ...
+```
+
+[![asciicast](https://asciinema.org/a/bahBYyfDyLoETR0qf1cQl7Stb.svg)](https://asciinema.org/a/bahBYyfDyLoETR0qf1cQl7Stb)
+
+By default, `Conda.jl` puts environments into `~/.julia/conda/envs/<env name>`,
+which you can get with `Conda.bin_dir()`, so in this case, you'd next want to run
+
+```@repl
+ENV["PATH"] = ENV["PATH"] * ":" * Conda.bin_dir(:biobakery)
```

Comment:
In the rendered tutorial, the next line after gives and error.
```
julia> ENV["PATH"] = ENV["PATH"] * ":" * Conda.bin_dir(:biobakery)
ERROR: UndefVarError: Conda not defined
```
I feel like this is not supposed to happen...

Reply:
Yeah - I'm switching everything over to static blocks instead of `@repl` (I started to try to do this dynamically, but `metaphlan` takes about 30 min to run the first time on my labtop, I don't want to have to do that everytime the docs build)

fixed in #69